deployments/local-setup-environment/provisioning/install-geth.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+echo "Installing go-ethereum..."
+
+GETH_PACKAGE=geth-alltools-linux-amd64-1.9.9-01744997.tar.gz
+
+curl -O https://gethstore.blob.core.windows.net/builds/$GETH_PACKAGE
```

Comment:
Why don't we use PPAs as recommended in `go-ethereum` docs? https://geth.ethereum.org/docs/install-and-build/installing-geth#install-on-ubuntu-via-ppas

Reply:
We need a specific version of Geth which is not provided by the PPA. Using the newest version caused problems with generated ABIs so I use `1.9.9` which works as expected.